From 275253d693c85dc12b385e547d5b26c2194b5aa1 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:03 -0500
Subject: [redhat] kabi: Adding symbol _dev_info (drivers/base/core.c)

Message-id: <d93ff4107e874f8dab0948fe7fff423e9bed9b3c.1418903403.git.poros@redhat.com>
Patchwork-id: 102245
O-Subject: [PATCH RHEL7.1 11/48] kabi: Adding symbol _dev_info (include/linux/device.h)
Bugzilla: 1160751
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160751
    https://bugzilla.redhat.com/show_bug.cgi?id=1160751

    Description:
    ============
    These are just conventient forms of printk(), if alternatives
    are considered preferable that isn't a big deal.
    Used in xpmem.ko

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7 BZ1062669 36/49] kabi: Adding symbol _dev_info (include/linux/device.h)
 Jiri Olsa <jolsa@redhat.com> (Fri, 14 Feb 2014 14:23:58 +0100)
 Description:
 ============
 These are just conventient forms of printk(), if alternatives
 are considered preferable that isn't a big deal.

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/_dev_info b/redhat/kabi/kabi-module/kabi_ppc64/_dev_info
new file mode 100644
index 0000000..19ba72a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/_dev_info
@@ -0,0 +1,2 @@
+#1-
+0x061d1859 _dev_info vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/_dev_info b/redhat/kabi/kabi-module/kabi_ppc64le/_dev_info
new file mode 100644
index 0000000..0d693ee
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/_dev_info
@@ -0,0 +1,2 @@
+#1-
+0x8a0e80b5 _dev_info vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/_dev_info b/redhat/kabi/kabi-module/kabi_s390x/_dev_info
new file mode 100644
index 0000000..03b58e7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/_dev_info
@@ -0,0 +1,2 @@
+#1-
+0x03aecf29 _dev_info vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/_dev_info b/redhat/kabi/kabi-module/kabi_x86_64/_dev_info
new file mode 100644
index 0000000..4db4d95
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/_dev_info
@@ -0,0 +1,2 @@
+#1-
+0x4cce8b07 _dev_info vmlinux EXPORT_SYMBOL
-- 
1.7.1